dbt_project.yml

```diff
@@ -1,5 +1,5 @@
 name: 'snowplow_web'
-version: '0.14.0-rc1'
+version: '0.14.0-rc2'
```

Comment:
The version in the custom-project's project.yml is still pointing at the former release.

Reply:
Good catch, I forgot there is 3 places in web...